api/grpcserver_test.go

```diff
@@ -2974,3 +2974,29 @@ func TestChainlinkErrTest(t *testing.T) {
 		})
 	}
 }
+
+func TestGrpcServer_TraceTransactionStructLogs(t *testing.T) {
+	require := require.New(t)
+	cfg := newConfig(t)
+
+	svr, bfIndexFile, err := createServerV2(cfg, true)
+	require.NoError(err)
+	defer func() {
+		testutil.CleanupPath(t, bfIndexFile)
+	}()
+
+	request := &iotexapi.TraceTransactionStructLogsRequest{
+		ActionHash: hex.EncodeToString(hash.ZeroHash256[:]),
+	}
+	_, err = svr.GrpcServer.TraceTransactionStructLogs(context.Background(), request)
+	require.Error(err)
+
+	//unsupport type
+	request.ActionHash = hex.EncodeToString(transferHash1[:])
+	_, err = svr.GrpcServer.TraceTransactionStructLogs(context.Background(), request)
+	require.Error(err)
+
+	request.ActionHash = hex.EncodeToString(executionHash1[:])
+	_, err = svr.GrpcServer.TraceTransactionStructLogs(context.Background(), request)
+	require.NoError(err)
+}
```

Comment:
Compare the detail of response with the expected

Reply:
done

Reply:
can you also add other fields of log?